docs/configuration_requirements.rst

```diff
@@ -0,0 +1,19 @@
+Configuration requirements
```

Comment:
I find the term "configuration requirements" not very intuitive (same for the variable name in irt and the occurence in the text below). I do have to confess I'm having trouble coming up with a good title. "Compatible versions" or "Version constraints" are the best I can think of right now, but I don't really like either of them very much. Will give this some more thought.

Reply:
What about:
Title : "Compatibility"
In text: "Compatible versions"

Reply:
Sounds OK to me. Might be a good idea to quickly demo this next week. If anyone doesn't like the section name we'll hear it then.